docker-compose-deploy.yml

```diff
@@ -57,6 +60,8 @@ services:
     ports:
       - 8000:8000
     command: bash -c "python manage.py migrate && python manage.py bootstrap && python manage.py collectstatic --noinput && gunicorn aml.wsgi:application --bind 0.0.0.0:8000"
+    restart: always
```

Comment:
I would expect that the depends_on conditions would also restart db / ip2country. Can you check if this is the case? Just wondering whether it could be potentially problematic if the db and ip2country service got a signal from the `server` to restart, while also getting their own signal to restart.